internal/parser/xml_parser.go

```diff
@@ -53,6 +99,84 @@ func ParseGroupAddressExport(filePath string) (*models.KNX, error) {
 	return &knxItems, nil
 }
 
+func readAndIDFile(filePath string) (FileID, []byte, error) {
```

Comment:
I don't really see the value in having the ability of parsing a file to try to identify its format. I prefer if we try to keep things simple. Assume format based on the file ending instead, `.xml` vs `.knxproj`. That way we can avoid this.